src/components/MultipleAvatars.js

```diff
@@ -117,20 +117,26 @@ const MultipleAvatars = (props) => {
                     style={singleAvatarStyles}
                 >
                     <Tooltip text={props.avatarTooltips[0]} absolute>
-                        <Image
-                            source={{uri: props.icons[0]}}
-                            style={singleAvatarStyles}
-                        />
+                        <View>
```

Comment:
`Avatar` already has a `View` wrapper - is this `View` needed? (same with below)

https://github.com/Expensify/App/blob/5a09253d1db8a3855dc1ca1f1cb5b3fdd23661fb/src/components/Avatar.js#L83

Reply:
I'll add a comment here, this View is necessary for the tooltip to show (multiple avatars/group chats in LHN) for some reason

Reply:
Interesting! I played around with removing it & trying some things, couldn't get the tooltip to display nicely without it 👍 Let's keep it! haha